indexing-service/src/test/java/org/apache/druid/indexing/overlord/TaskQueueTest.java

```diff
@@ -443,9 +461,8 @@ public void testKilledTasksEmitRuntimeMetricWithHttpRemote() throws InterruptedE
 
     // Verify that metrics are emitted on receiving announcement
     serviceEmitter.verifyEmitted("task/run/time", 1);
-    CoordinatorRunStats stats = taskQueue.getQueueStats();
-    Assert.assertEquals(0L, stats.get(Stats.TaskQueue.STATUS_UPDATES_IN_QUEUE));
-    Assert.assertEquals(1L, stats.get(Stats.TaskQueue.HANDLED_STATUS_UPDATES));
+    verifyTaskStatusCounts(taskQueue, 0, 1);
+    verifyStatusUpdateCounts(taskQueue, 0, 2);
```

Comment:
To make this more readable, please invoke the verify methods separately for each metric like:

```
verifyMetric(metricName, expectedValue);
```